backend/src/mailing/mailing.controller.ts

```diff
@@ -1,26 +1,34 @@
-import { Controller, Post, Body } from '@nestjs/common';
+import { Controller, Post, Body, UseGuards } from '@nestjs/common';
 import { MailingService } from './mailing.service';
 import {
   ContactRequestDto,
   ContactResponseDto,
   DefaultEmailResponseDto,
   DefaultEmailDto,
 } from './mailing.dto';
-import { Public } from 'src/auth/decorators/user-level.decorator';
+import { UserLevels } from '../auth/decorators/user-level.decorator';
+import { UserLevelGuard } from '../auth/guards/user-level.guard';
+import { JwtAuthGuard } from '../auth/guards/jwt-auth.guard';
+import { UserLevel } from '@prisma/client';
+import { ApiBearerAuth } from '@nestjs/swagger';
 
 @Controller('mailing')
+@UseGuards(JwtAuthGuard, UserLevelGuard)
 export class MailingController {
   constructor(private readonly mailingService: MailingService) {}
 
-  @Public()
   @Post('/contact')
+  @UserLevels(UserLevel.Superadmin, UserLevel.Admin, UserLevel.Default)
```

Comment:
Essa rota serve de apoio para o painel de contato na home do portal, na minha visão deveria possuir acesso público de modo a não exigir que um membro da comunidade se registre para tirar dúvidas, por exemplo.
```suggestion
  @Public()
```